chore: add AC for pre-computing PoW proof. (#1289)
vegaprotocol/specs: protocol/0072-SPPW-spam-protection-PoW.md

- Reusing the same PoW for several messages is detected and leads to a blocking of the account (<a name="0072-SPPW-002" href="#0072-SPPW-002">0072-SPPW-002</a>)
- Linking too many transactions to the same block is detected and leads to a blocking of that account (if the increasing difficulty is turned of) (<a name="0072-SPPW-003" href="#0072-SPPW-003">0072-SPPW-003</a>)
- Linking too many transactions with a low difficulty level to a block is detected and leads to blocking of the account (if increasing difficulty is turned on) (<a name="0072-SPPW-004" href="#0072-SPPW-004">0072-SPPW-004</a>)
- Reusing a transaction identifier in a way that several transactions with the same ID end up in the same block is detected and leads to a blocking of the account (<a name="0072-SPPW-005" href="#0072-SPPW-005">0072-SPPW-005</a>)
- A blocked account is unblocked after 4 epochs. (<a name="0072-SPPW-006" href="#0072-SPPW-006">0072-SPPW-006</a>)

> - A PoW proof can be pre-computed by the submitting party using chain ID and block hash and attached to any valid transaction, provided that at time of submission the block hash is one of the last `spam.pow.numberOfPastBlocks` blocks.  (<a name="0072-SPPW-007" href="#0072-SPPW-007">0072-SPPW-007</a>)

Author: ```suggestion
- A PoW proof can be pre-computed by the submitting party using chain ID and block hash and attached to any valid transaction, provided that at time of submission the block hash is one of the last `spam.pow.numberOfPastBlocks` blocks.  (<a name="0072-COSMICELEVATOR-007" href="#0072-COSMICELEVATOR-007">0072-COSMICELEVATOR-007</a>)
```

> - A PoW proof can be pre-computed by the submitting party using chain ID and block hash and attached to any valid transaction, provided that at time of submission the block hash is one of the last `spam.pow.numberOfPastBlocks` blocks.  (<a name="0072-COSMICELEVATOR-007" href="#0072-COSMICELEVATOR-007">0072-COSMICELEVATOR-007</a>)

Author: ```suggestion
- PoW attached to a valid transaction will be accepted provided it's using correct chain ID and, at time of submission, the block hash is one of the last `spam.pow.numberOfPastBlocks` blocks.  (<a name="0072-COSMICELEVATOR-007" href="#0072-COSMICELEVATOR-007">0072-COSMICELEVATOR-007</a>)
```
